Manager/src/main/java/io/raspberrywallet/manager/linux/TemperatureMonitor.java

```diff
@@ -20,26 +20,26 @@ public TemperatureMonitor() {
      */
 
     @Override
-    public String run() {
+    public String call() {
         try {
             Process process = Runtime.getRuntime().exec("cat /sys/class/thermal/thermal_zone0/temp");
             process.waitFor();
-            
+
             return parseTemperatureFrom(process);
-            
+
         } catch (IOException | InterruptedException e) {
             e.printStackTrace();
             return "error";
         }
     }
-    
+
     private String parseTemperatureFrom(Process process) throws IOException {
         try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
```

Comment:
Mock this to test.

Reply:
It should be tested in `TemperatureMonitorTest`, not `ManagerTest`

Reply:
as Stasbar said, we cannot test things in more than one place, it will make our code unchangeable